File: app/src/main/java/com/callmangement/ui/errors/model/UpdateErrorStatusResponse.java
package com.callmangement.ui.errors.model;

public class UpdateErrorStatusResponse {
    public boolean status;
    public String message;
    public String ErrorRegNo;


    public UpdateErrorStatusResponse(boolean status, String message, String errorRegNo) {
        this.status = status;
        this.message = message;
        ErrorRegNo = errorRegNo;
    }


    public boolean isStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getErrorRegNo() {
        return ErrorRegNo;
    }

    public void setErrorRegNo(String errorRegNo) {
        ErrorRegNo = errorRegNo;
    }
}

File: app/src/main/java/com/callmangement/ui/errors/model/UpdateErrorStatusResponse.kt
package com.callmangement.ui.errors.model

import com.google.gson.annotations.SerializedName

class UpdateErrorStatusResponse(
    var isStatus: Boolean, @JvmField var message: String, @field:SerializedName(
        "ErrorRegNo"
    ) var errorRegNo: String
)
